src/Pages/RoomLists/Components/StatusLabel/StatusLabel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import Theme from '../../../../Styles/Theme';
+
+const StatusLabel = ({ color, text }) => {
+  return <Label color={color}>{text}</Label>;
+};
+
+export default StatusLabel;
+
+const Label = styled.span`
+  ${Theme.flexCenter};
+  width: 38px;
```

Comment:
```suggestion
import styled from 'styled-components';

const StatusLabel = ({ color, text }) => {
  return <Label color={color}>{text}</Label>;
};

export default StatusLabel;

const Label = styled.span`
  ${props => props.theme.flexCenter};
  width: 38px;
```
theme 객체는 themeProvider에서 꺼내어 사용하면 됩니다!

Reply:
아..다 수정해야겠군요 수정하겠습니다.!